scripts/contracts_add_export.js

```diff
@@ -4,7 +4,7 @@
 const fs = require('fs')
 const path = require('path')
 
-const file = path.resolve(__dirname, '../src/custom/abis/types/index.ts')
+const file = path.resolve(__dirname, '../src/cow-react/abis/types/index.ts')
```

Comment:
I am blind, I'm sorry

Reply:
its fine :) 

actually was a good thought u had, i could have easily missed that if it wasn't for the app not booting without that change (it was not finding the ERC20 contract)